notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">radius = "0.1" #[deg] = 6 arcminutes
</pre><p>Could you add a comment why 0.1 degree was used?</p><p> </p><pre class="ql-syntax" spellcheck="false">condition = "nDetections.gte=1"
</pre><p>Please add some comment about what this condition implies.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>It's not still clear how radius was chosen though.</p>